doozerlib/dblib.py

```diff
@@ -107,8 +107,8 @@ def check_missing_db_env_var(self):
         """
 
         if not (constants.DB_USER in os.environ and constants.DB_PWD in os.environ):
-            self.runtime.logger.error("Environment variables required for db operation missing. Doozer will be running"
-                                      "in no DB use mode.")
+            self.runtime.logger.warning("Environment variables required for db operation missing. Doozer will be running"
+                                        "in no DB use mode.")
```

Comment:
the change is fine, but it's better to make a separate commit for unrelated nits like this.